src/content/h1/info/readme.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hard-coded Data
+keywords:
+  - color
+  - data
+  - engine
+  - hard-coded
+  - info
+thanks:
+  - to: MosesOfEgypt
+    for: Armor color values
+  - to: Lavadeeto
+    for: Armor color testing and picture
+---
+There are lots of hard-coded values in Halo's engine, such as the multiplayer
+armor color. These values do not typically appear anywhere in tag data, and
+cannot be directly changed or controlled.
+
+# Multiplayer Armor Colors
+
+<figure>
+  <a href="armor_colors.png">
+    <img src="armor_colors.png" alt="Armor colors in order"/>
+  </a>
+  <figcaption>
+    Spartan armor colors with shaders as they appear ingame.
+    The far-left is the single-player color.
```

Comment:
Where does the singleplayer colour come from? Should this say something like "the far-left is the single-player color, which is not a hardcoded value and comes from the biped's X tag"?